system_report.sh

```diff
@@ -45,6 +45,7 @@ ver_line="$(zip -v | head -n 2 | tail -n 1)";     echo "* zip: $ver_line"
 ver_line="$(tar --version | head -n 1)" ;         echo "* tar: $ver_line"
 ver_line="$(tree --version)" ;                    echo "* tree: $ver_line"
 ver_line="$(gcc --version | head -n 1)" ;         echo "* gcc: $ver_line"
+ver_line="$(convert -version | grep Version)" ;   echo "* ImageMagick: $ver_line"
```

Comment:
Could you please change this to https://github.com/bitrise-io/osx-box-bootstrap/blob/master/system_report.sh#L64 ?

Sorry, I forgot to say that we have a version report for IM in the Mac provisioning repo..